src/main/java/ay2122s1_cs2103t_w16_2/btbb/logic/commands/order/FindOrderCommand.java

```diff
@@ -27,6 +29,8 @@
             + "[" + PREFIX_CLIENT_NAME + "CLIENT_NAME] "
```

Comment:
Could you edit the top part of the message too?
`client name, client phone number, client address, recipe name, order date and order status fields.`